advanced/editor-command-identifiers.md

```diff
@@ -500,16 +502,19 @@ The following command values can be queried using the [queryCommandValue]({{ sit
 
 | Command       | Description                                            |
 | --------------| ------------------------------------------------------ |
-| FontName      | Returns the font name of the current selection.        |​
-| FontSize      | Returns the font size of the current selection.        |​
+| FontName      | Returns the font name of the current selection.        |
+| FontSize      | Returns the font size of the current selection.        |
 | ToggleSidebar | Returns the current state of sidebar (open or closed). |
+| LineHeight    | Returns the line height of the current selection.      |
```

Comment:
Maybe put this above, so it's in alphabetical order?

Reply:
will do. I also just realised that this doesn't have a "5.5 only" warning